Ptt/AppDelegate.swift

```diff
@@ -32,6 +32,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             window.makeKeyAndVisible()
 
         }
+        if #available(iOS 15, *) {
+            let appearance = UINavigationBarAppearance()
+            appearance.configureWithDefaultBackground()
+            UINavigationBar.appearance().standardAppearance = appearance
+            UINavigationBar.appearance().compactAppearance = appearance
+            UINavigationBar.appearance().scrollEdgeAppearance = appearance
+        }
```

Comment:
I have yet another suggestion: move these to view controllers in need, like:
`navigationController?.navigationBar.standardAppearance = appearance`

rather than put it as global appearance, so we can still enjoy the default design benefits of iOS 15, as screenshot below.

Also, [FBPageViewController](https://github.com/Ptt-official-app/Ptt-iOS/blob/main/Ptt/Flows/FBPageFlow/Controllers/FBPageViewController.swift) uses web view and has similar issue, too. Would you like to help fix it as well? Or I'll do it later :)

![Navigation bar design before and after iOS 15](https://user-images.githubusercontent.com/2727287/145831332-cc2b5165-bc78-4848-ae6e-8f64c1f4a30c.jpg)

Reply:
Done, create a `NavigationController` extension.    

Do we need the FB page?  
As a user, I think a button on the Settings page is enough.   
The UX of Facebook mobile web page is not good, especially when it is not login      

Furthermore, reviewing the Facebook page is not the main purpose that users open a ptt app